interfaces/builtin/mount_observe.go

```diff
@@ -39,11 +39,17 @@ const mountObserveConnectedPlugAppArmor = `
 
 # Needed by 'df'. This is an information leak
 @{PROC}/mounts r,
+# Needed by 'htop' to detect whether it's running under lxc/lxd/docker
+@{PROC}/1/mounts r,
+
 owner @{PROC}/@{pid}/mounts r,
 owner @{PROC}/@{pid}/mountinfo r,
 owner @{PROC}/@{pid}/mountstats r,
 /sys/devices/*/block/{,**} r,
 
+# Needed by 'htop' to calculate RAM usage more accurately (and informational purposes, if enabled)
```

Comment:
Would this be more appropriate under `system-observe` as it doesn't seem mount related to my understanding?

Reply:
That's a good point.